### PR TITLE
chore(.github): create PR checklist

### PR DESCRIPTION
**Pull request by karansapolia**
This PR creates a PR checklist for all subsequent PRs submitted to the project.
fixes: #36

### CODE REVIEW

**Comment by piyushagwl007** [comment]
@karansapolia Please update the branch with dev

**Comment by piyushagwl007** [comment]
@karansapolia This has a dependency on #34. Let's wait for it to be merged.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## PR Checklist
+Please check that your PR fulfills the following requirements:
+
+- [ ] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
```
Comment (piyushagwl007) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: LGTM just one change, should have 1 more checkbox for linting passes.

**Review by piyushagwl007 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## PR Type
+What kind of change does this PR introduce? 
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
```
Comment (nihalanisumit) [inline .github/PULL_REQUEST_TEMPLATE.md]: don't leave space

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (nimish-gupta) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Also for gifs/videos for frontend PRs

**Review by nimish-gupta — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (karansapolia) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: > LGTM just one change, should have 1 more checkbox for linting passes.

That will not be needed, as circle CI will have the yarn lint tests once eslint PR is pushed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (karansapolia) [reply .github/PULL_REQUEST_TEMPLATE.md]: Could you be specific :slightly_smiling_face:

**Review by nihalanisumit — APPROVED** [review]
LGTM

**Pull request merged**